Mathlib/Topology/Algebra/ClopenNhdofOne.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2024 Nailin Guan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nailin Guan, Yi Song, Xuchun Li
+-/
+import Mathlib.GroupTheory.Index
+import Mathlib.Topology.Algebra.ClosedSubgroup
+import Mathlib.Topology.Algebra.OpenSubgroup
+/-!
+## The Open Normal Subgroup in a Clopen Neighborhood of One
+This file define `OpenNormalSubgroupSubClopenNhdsOfOne` , which strengthen the existing result of
+`OpenSubgroupSubClopenNhdsOfOne` into open normal subgroup.
+
+This file is split out because the need of more imports.
+-/
+
+namespace TopologicalGroup
+
+/-- The open normal subgroup contained in a clopen nhd of `1` in a compact topological group. -/
+noncomputable def OpenNormalSubgroupSubClopenNhdsOfOne {G : Type*} [Group G] [TopologicalSpace G]
```

Comment:
Can we use `@[to_additive]` to generate the analogous result for additive topological groups?

Reply:
I think for this one, we can't, because `NormalCore` doesn't have an additive version, or I should add one myself?

Reply:
Yes, that should be additivized to `AddNormalCore` or something like that.

Reply:
`@[to_additive]` failed for the `AddNormalCore`, and I think it might need quite a few changes. So should I start a new PR about this or put it just in the current PR?

Reply:
Ok, let's not do that in this PR.